Fix triton_reshape to properly expand `Min` keyword in triton codegen

arui-meta opened the pull request:
Summary: Previously triton_reshape will generate code with `Min` keyword in it, which is incorrect. This diff updates the triton_reshape function to properly expand `Min` keyword to `<`.

Test Plan:
```
buck2 run @//mode/{opt,mtia,inplace} //glow/fb/fx/fba/tests:test_fba_inductor -- -r test_Min_keyword_in_block_shape
```

Differential Revision: D63850158




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

arui-meta commented:
@pytorchbot label "topic: not user facing"

blaine-rister reviewed (approved):
LGTM! Thanks for the fix.

arui-meta committed:
Fix triton_reshape to properly expand Min keyword in triton codegen (#137357)

Summary:

Previously triton_reshape will generate code with `Min` keyword in it, which is incorrect. This diff updates the triton_reshape function to properly expand `Min` keyword to `<`.

Test Plan:
```
buck2 run @//mode/{opt,inplace} fbcode//caffe2/test/inductor:torchinductor_strided_blocks -- -r test_complex_reshape_block_ptr
```

Reviewed By: blaine-rister

Differential Revision: D63850158
